lualib/modular_admin/modular_admin_tag.lua

```diff
@@ -72,7 +72,7 @@ function modular_admin_tag_gui_clicked(event)
 	local e = event.element
 	if e ~= nil then
 		if p.admin then
-			if e.parent.name == modular_admin_get_menu(p).name then
+			if e.parent ~= nil and (e.parent.name == modular_admin_get_menu(p).name) then
```

Comment:
e.parent ~= nil can be earlier at the e.valid check.

Reply:
I'm assuming that this is the right code to begin with, and given my lack of knowledge I assumed it would make sense to have it next to the parent part. If it should be moved higher then be my guest.